authorization/src/main/kotlin/com/ritense/authorization/web/rest/PermissionManagementResource.kt

```diff
@@ -32,9 +33,10 @@ class PermissionManagementResource(
     val permissionRepository: PermissionRepository
 ) {
     @PostMapping("/v1/permissions/search")
-    @JsonView(PermissionView.PermissionManagement::class)
-    fun searchPermissions(@RequestBody searchRequest: SearchPermissionsRequest): ResponseEntity<List<Permission>> {
-        val rolePermissions = permissionRepository.findAllByRoleKeyInOrderByRoleKeyAscResourceTypeAsc(searchRequest.roles)
+    fun searchPermissions(@RequestBody searchRequest: SearchPermissionsRequest): ResponseEntity<List<PermissionDto>> {
+        val rolePermissions = permissionRepository
```

Comment:
I know it was like that before your change, but it's generally bad practice to use the repositories from the resource/controller.